core/shared/src/main/scala/com/monovore/decline/HelpFormat.scala

```diff
@@ -0,0 +1,27 @@
+package com.monovore.decline
+
+sealed abstract class HelpFormat extends Product with Serializable {
+  def colorsEnabled: Boolean
+}
+object HelpFormat {
+  case object Plain extends HelpFormat {
+    override def colorsEnabled: Boolean = false
+  }
+  case object Colors extends HelpFormat {
+    override def colorsEnabled: Boolean = true
+  }
+  case class AutoColors(env: Map[String, String] = sys.env) extends HelpFormat {
```

Comment:
Neat - was not aware of this!

This could be a method returning a format instead of a format of its own... and I'd prefer to not include `sys.env` as a default argument, since it's arguably not pure.